### PR TITLE
Client: Building blocks for Entity Change Notifications

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: systems/WikibaseClient/diagrams/05-entitychangenotifications-usage.drawio.svg [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+                    <div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 108px; height: 1px; padding-top: 30px; margin-left: 536px;">
+                        <div style="box-sizing: border-box; font-size: 0; text-align: center; ">
+                            <div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">
+                                Huh?
+                            </div>
+                        </div>
+                    </div>
+                </foreignObject>
+                <text x="590" y="35" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">
+                    Huh?
```
Comment (jakobw) [inline systems/WikibaseClient/diagrams/05-entitychangenotifications-usage.drawio.svg:277]: Looks a bit funny to keep it here, but combined with the warning it's probably ok as an indicator of something that needs to be cleaned up :D

**Review by jakobw — APPROVED** [review]
I like the links from the higher level diagram to the lower-level ones!! :)

**Review by tzhelyazkova — CHANGES_REQUESTED** [review]
All looks good but if we are keeping `UsageAspectTransformer`, the block context shouldn't say "Huh?"

**Comment by addshore** [comment]
> All looks good but if we are keeping `UsageAspectTransformer`, the block context shouldn't say "Huh?"
> I think it'd be better if we remove the purple "Huh" all together and leave `UsageAspectTransformer` just on the side.

I'll do that!

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**